src/index.js

```diff
@@ -0,0 +1,38 @@
+import './style.css';
+
+const listItems = document.querySelector('.listItems');
+
+const task = [
+  {
+    description: 'Study chapter 1',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'Study chapter 2',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Study chapter 3',
+    completed: false,
+    index: 2,
+  },
+];
```

Comment:
- [ ] **Required:** The list should be displayed in the order of the indexes. As the code is at the moment if the index value is arranged unorderly, it has no effect. 

**Hint:** Use the sort function to select the order in which they are displayed. For example, `Array.sort((a, b) => a.index - b.index)`

Reply:
I added it, thank you so much for the hint :smiley: